build/tests.webpack.js

```diff
@@ -1,3 +1,3 @@
-var context = require.context('../web', true, /(-test\.jsx?)|(-test-chrome\.jsx?)$/);
+var context = require.context('../web/client/components/catalog/editor/__tests__', true, /(-test\.jsx?)|(-test-chrome\.jsx?)$/);
```

Comment:
You should not commit this
```suggestion
var context = require.context('../web', true, /(-test\.jsx?)|(-test-chrome\.jsx?)$/);
```